app/lib/util/files.js

```diff
@@ -110,35 +116,54 @@ function readFileAsJSON(file) {
 }
 
 /**
- * Find files by pattern in a list of search paths.
+ * Find files matching a given pattern in one or more search paths.
  *
- * @param {Object} options
- * @param {Array<string>} options.searchPaths
- * @param {string} options.pattern
+ * @param {string} pattern
+ * @param {Object} [options]
+ * @param {string[]} [options.searchPaths]
+ * @param {boolean} [options.absolute=true]
+ * @param {boolean} [options.nodir=true]
```

Comment:
Not sure where we use `nodir`. I believe `onlyFiles` is the fast-glob thing?

Reply:
We used `nodir` in the past. I just forgot to change it in the JSDoc.

Reply:
Fixed via https://github.com/camunda/camunda-modeler/pull/3863/commits/b82fbb55b5cca92019f6f75602817d414035b192.